Source/JavaScriptCore/runtime/ISO8601.h

```diff
@@ -277,9 +279,9 @@ std::optional<int64_t> parseUTCOffset(StringView, bool parseSubMinutePrecision =
 std::optional<int64_t> parseUTCOffsetInMinutes(StringView);
 enum class ValidateTimeZoneID : bool { No, Yes };
 std::optional<std::tuple<PlainTime, std::optional<TimeZoneRecord>>> parseTime(StringView);
-std::optional<std::tuple<PlainTime, std::optional<TimeZoneRecord>, std::optional<CalendarRecord>>> parseCalendarTime(StringView);
+std::optional<std::tuple<PlainTime, std::optional<TimeZoneRecord>, std::optional<Vector<CalendarRecord>>>> parseCalendarTime(StringView);
```

Comment:
Most common case is one-calendar when exists. So use `Vector<CalendarRecord, 1>` throughout the code.

Reply:
I'm actually thinking it'd be better not to return a vector here. If there are two calendar annotations and neither are critical, the spec says the first one must be chosen: https://www.rfc-editor.org/rfc/rfc9557.html#section-3.3-16

Reply:
Fixed in 51b4125c. I used `Vector<CalendarRecord, 1>` in this function, but switched back to `std::optional<CalendarRecord>` at callers, because if we parse ≥2 calendars we should ignore all subsequent ones.